util/dvsim/LsfLauncher.py

```diff
@@ -91,7 +91,7 @@ def make_job_script(cfg, job_name):
         lines += ["case $1 in\n"]
         for job in LsfLauncher.jobs[cfg][job_name]:
             # Redirect the job's stdout and stderr to its log file.
-            cmd = "{} > {} 2>&1".format(job.deploy.cmd,
+            cmd = "{} > {} 2>&1".format(" ".join(job.deploy.cmd),
```

Comment:
This will need to be the quoted form (`" ".join(shlex.quote(arg) for arg in self.deploy.cmd)`). If this isn't obvious, it might be worth reading up on how shell expansion works.